lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -352,10 +369,10 @@ def inner(self, default_store, module_count, mongo_calls, sql_queries, *args, **
         return inner
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 3, 4, 22),
-        (ModuleStoreEnum.Type.mongo, 20, 4, 22),
-        (ModuleStoreEnum.Type.split, 3, 13, 22),
-        (ModuleStoreEnum.Type.split, 20, 13, 22),
+        (ModuleStoreEnum.Type.mongo, 3, 4, 23),
```

Comment:
Really it is looking up the user that is causing the extra query, correct? We don't update last_activity_at unless the user is in a team.